product-service/src/lambdas/getProductById.ts

```diff
@@ -0,0 +1,20 @@
+import { ProductsService, ProductInterface } from "../../libs/services/products.service";
+import { errorResponse, successResponse, responseInterface } from "../../libs/response-helpers";
```

Comment:
this is utility

Reply:
src/getProductById.ts simply this

Lambda folder reedundant

Reply:
Fixed it on 53cb972